CHANGELOG.md

```diff
@@ -1,3 +1,338 @@
+
+  If you are running Lokomotive on Equinix Metal (formerly Packet), then install Packet CCM. Export your Packet cluster's project ID and API Key.
+
+  ```bash
+  export PACKET_AUTH_TOKEN=""
+  export PACKET_PROJECT_ID=""
+
+  echo "apiKey: $PACKET_AUTH_TOKEN
+projectID: $PACKET_PROJECT_ID" > /tmp/ccm-values.yaml
+
+  helm install packet-ccm --namespace kube-system --values=/tmp/ccm-values.yaml ./lokomotive-0.6.0/assets/charts/control-plane/packet-ccm/
+  ```
+
+3. Update node config.
+
+  This script shipped with release tarball will add permanent MetalLB labels and kubelet config to use CCM.
+
+  > **NOTE**: Please edit this script to disable updating certain nodes. Modify `update_other_nodes` function as required.
+
+  ```bash
+  ./lokomotive-0.6.0/scripts/update/0.5-0.6/update.sh false
+  ```
+
+4. Reload the new config for self-hosted kubelet.
+
+  ```bash
+  kubectl -n kube-system rollout restart ds kubelet
+  ```
+
+  If the above command failed, probably you are not running self-hosted kubelet, which is fine because the script in step 3 already updated the host kubelet.
+
+5. Export assets directory.
+
+  ```bash
+  export ASSETS_DIR="assets"
+  ```
+
+6. Remove BGP sessions from Terraform state.
+
+  If you are running Lokomotive on Equinix Metal (formerly Packet), then run the following commands:
+
+  ```bash
+  cd $ASSETS_DIR/terraform
+  terraform state rm $(terraform state list | grep packet_bgp_session.bgp)
+  cd -
+  ```
+
+7. Remove old asset files.
+
+  ```bash
+  rm -rf $ASSETS_DIR/cluster-assets
+  rm -rf $ASSETS_DIR/terraform-modules
+  ```
+
+8. Update control plane.
+
+  ```bash
+  lokoctl cluster apply --skip-components -v
+  ```
+
+  > **NOTE:** If the update process gets interrupted, rerun above command.
+
+  > **NOTE:** If you are running self-hosted kubelet then append the above command with flag `--upgrade-kubelets`.
+
+  The update process typically takes about 10 minutes.
+  After the update, running `lokoctl health` should result in an output similar to the following:
+
+  ```bash
+  Node                     Ready    Reason          Message
+
+  lokomotive-controller-0  True     KubeletReady    kubelet is posting ready status
+  lokomotive-1-worker-0    True     KubeletReady    kubelet is posting ready status
+  lokomotive-1-worker-1    True     KubeletReady    kubelet is posting ready status
+  lokomotive-1-worker-2    True     KubeletReady    kubelet is posting ready status
+  Name      Status    Message              Error
+
+  etcd-0    True      {"health":"true"}
+  ```
+
+9. Update the bootstrap components: kubelet and etcd.
+
+  This script shipped with release tarball will update all the nodes to run the latest kubelet and etcd.
+
+  > **NOTE**: Please edit this script to disable updating certain nodes. Modify `update_other_nodes` function as required.
+
+  ```bash
+  ./lokomotive-0.6.0/scripts/update/0.5-0.6/update.sh true
+  ```
+
+10. Reload the new config for self-hosted kubelet.
```

Comment:
Do we need to run this twice? We've run it already in step 4.

Reply:
If we need to run it twice, please change the text as mentioned in the previous comment.

Reply:
This time the reload is to update the version of kubelet. Earlier one was to update a piece of config.